### DISCUSSION TITLE
the command typegen to generate type does not work

### DISCUSSION THREAD

**Discussion by ericksonholguin** [chakra-ui/chakra-ui#9169] [General] [answered]
### Description

Hello, I'm trying to create custom breakpoints that I need in my project, but I am faced with the following problem:

![image](https://github.com/user-attachments/assets/7c64e6f6-c62a-415b-93c8-48729c2ff7fd)

This is my code: 
<img width="965" alt="Screenshot 2024-11-15 at 6 29 10 AM" src="https://github.com/user-attachments/assets/df4dccb7-b796-4f26-9743-275ac036823d">

<img width="1554" alt="Screenshot 2024-11-15 at 6 30 00 AM" src="https://github.com/user-attachments/assets/4441b153-a86e-4dff-98ce-402c8863231d">

And I executed typegen in both possible ways using `chakra typegen` and `npx @chakra-ui/cli typegen` but I can't resolve the problem.
<img width="1551" alt="Screenshot 2024-11-15 at 6 30 40 AM" src="https://github.com/user-attachments/assets/114458de-af0c-42c0-8e95-637957eafd19">

In this discution: https://github.com/chakra-ui/chakra-ui/discussions/9016 someone told that we must to restart the TS server, I tried, but I keep seeing the problem. 

### Link to Reproduction

https://stackblitz.com/edit/chakra-ui-v3?file=src%2FApp.tsx

### Steps to reproduce

2. Click on '...'
3. Scroll down to '...'

### Chakra UI Version

3.1.1

### Operating System

- [X] macOS
- [ ] Windows
- [ ] Linux

**Comment by Arpan1089** [comment]
(Chakra typegen ./src/components/ui/theme.ts). Make sure it successfully updated type definitions.

**Comment by ericksonholguin** [comment]
> (Chakra typegen ./src/components/ui/theme.ts). Make sure it successfully updated type definitions.

That's not a solution, it's just what I'm doing and I detailed it in the description.

**Comment by segunadebayo** [comment, marked as answer]
@Eriickson I tried this locally, and it works correctly.

> I noticed those files you have in the screenshot look unsaved. Could you try saving those files?

Here's a video that proves that
https://github.com/user-attachments/assets/a9ba965f-d87a-4e6d-8d16-bb7aa7b9fe8d

**Reply by ericksonholguin** [reply]
I think that the problem was I had installed chackra cli globally and it isn't intalled in the project

**Reply by sayshark75** [reply]
Can you help me in this, if it is resolved for you?
- i added the chakra-cli in the project itself as dev dependency, and then run the cli command to create types, but no use, same issues.
- restarted typescript server. still same issue.

**Reply by sayshark75** [reply]
@segunadebayo, what is the `chakra-config.json` file holds? not generated in my project yet.

**Reply by koroman17** [reply]
I have the same issue

https://github.com/user-attachments/assets/1e5ec95d-461f-424a-a8fe-9e90e5dcfc5b

![image](https://github.com/user-attachments/assets/33989157-b4d2-4c43-a832-5b4373d839f1)

After installing a brand new project, it stoped bugging now. But in the bugged project, even though I tried to delete the node_modules and reinstall, it didn't show warnings at first but not after typegenning.
